tests/test_field_data.py

```diff
@@ -6,6 +6,7 @@
 from ansys.fluent.core.services.field_data import (
     ScalarFieldNameError,
     ScalarFieldUnavailable,
+    SurfaceDataType,
     SurfaceNameError,
 )
 
```

Comment:
@prmukherj Doc must be updated.

Reply:
Updated the doc. @PipKat could you please have a look. Thank you.

Reply:
@prmukherj Just a few minor edits--and questions to ensure that any suggestions have the meaning that you intended!

Reply:
Thank you.